Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -168,44 +179,44 @@ done < netdev.log
 		Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "./test.sh" -runAsSudo -RunInBackground -ignoreLinuxExitCode:$true | Out-Null
 		Write-LogInfo "Sent hibernate command to the VM and continue checking its status in every 15 seconds until 2 minutes timeout "
 
-		# Verify the VM status
-		# Can not find if VM hibernation completion or not as soon as it disconnects the network. Assume it is in timeout.
-		$timeout = New-Timespan -Minutes 10
-		$sw = [diagnostics.stopwatch]::StartNew()
-		while ($sw.elapsed -lt $timeout){
-			Wait-Time -seconds 15
-			$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
+			# Verify the VM status
+			# Can not find if VM hibernation completion or not as soon as it disconnects the network. Assume it is in timeout.
+			$timeout = New-Timespan -Minutes 20
+			$sw = [diagnostics.stopwatch]::StartNew()
+			while ($sw.elapsed -lt $timeout){
+				Wait-Time -seconds 15
+				$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
+				if ($vmStatus.Statuses[1].DisplayStatus -eq "VM stopped") {
+					break
+				} else {
+					Write-LogInfo "VM status is not stopped. Wating for 15 seconds..."
+				}
+			}
 			if ($vmStatus.Statuses[1].DisplayStatus -eq "VM stopped") {
-				break
+				Write-LogInfo "$($vmStatus.Statuses[1].DisplayStatus): Verified successfully VM status is stopped after hibernation command sent"
 			} else {
-				Write-LogInfo "VM status is not stopped. Wating for 15s..."
+				Write-LogErr "$($vmStatus.Statuses[1].DisplayStatus): Could not find the VM status after hibernation command sent"
+				throw "Can not identify VM status after hibernate"
```

Comment:
the VM status is not expected?

Reply:
Sometimes from Azure or VM does not change the state properly or in-time. Fixed the wording.

Reply:
Fixed in 389c7fdf2b96277a1b0c34ee9bbf34441c63dbec